troposphere/static/js/actions/InstanceActions.js

```diff
@@ -25,7 +25,7 @@ export default {
     redeploy: IRedeploy.redeploy,
     poll: IPoll.poll,
     launch: ILaunch.launch,
-    destroy: IDestroy.destroy,
+    destroy,
```

Comment:
So in a way, all imports such as `IUpdate`, `IReport`, etc. can be imported as, for example

```python
import { update } from "./instance/update";
```